src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -0,0 +1,131 @@
+package com.google.sps.servlets;
+
+import static com.google.sps.utils.ServletUtils.getParameter;
+import static com.google.sps.utils.StringConstants.*;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+import com.google.sps.utils.ServletUtils;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet for obtaining the Calendar ID of a group and adding or obtaining events
+ */
+@WebServlet("/events")
+public class EventsServlet extends AbstractEventsServlet {
+  private final int MAX_EVENTS = 50;
```

Comment:
Any reason for 50? e.g. is this just a number that you picked, or are there other limitations that make anything higher than 50 not great?

Reply:
No specific limitations saying no more than 50, we just thought that anything over 50 events on the map per group was definitely overkill and would slow everything down so we just thought it would be a good number.

Reply:
OK, could you add it in your doc and/or add a comment saying that you're capping it at 50 to not overwhelm the UI?